app/controllers/people/person/positions/minister/edit.js

```diff
@@ -0,0 +1,51 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { dropTask } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+import { combineFullAddress } from 'frontend-contact-hub/models/address';
+
+const FINANCING_CODE = {
+  SELF_FINANCED: '997073905f839ac6bafe92b76050ab0b',
+  FOD_FINANCED: '9d6f49b3d923b437ec3a91e8b5fa6885',
+};
+
+export default class PeoplePersonPositionsMinisterEditController extends Controller {
+  @service router;
+
+  @tracked willReceiveFinancing;
```

Comment:
At the moment this will always check the "no" radio button because `undefined` is a falsy value.
Th
is needs to be set to a value when the page is loaded. So adding a "setup" method which compares the `minister.financing.id` to the `FINANCING_CODE.FOD_FINANCED` might be a good enough solution for now.